src/views/CardStack.js

```diff
@@ -207,11 +224,16 @@ class CardStack extends React.Component<DefaultProps, Props, void> {
           if (header && header.left) {
             return header.left;
           }
-          const { renderLeftComponent } = this.props.headerComponent.defaultProps || {};
-          if (typeof renderLeftComponent === 'function') {
-            return renderLeftComponent(props);
+          if (props.scene.index === 0 || !props.onNavigateBack) {
+            return null;
           }
-          return null;
+          const prevSceneTitle = this._getHeaderTitleForScene(props.scenes[props.index - 1]);
+          return (
+            <HeaderBackButton
```

Comment:
> If you define a custom header, the this.props.renderLeftComponent(props) will return you a standard back button anyway as long as your route doesn't override it.

yeah, but it's the standard back button for old header. if I have my default left component defined on my header, it won't work.

> If you want to show a custom header button (left), it's already up to you to access the title yourself I believe.

cardstack reads the config and passes the left component.

> Not sure I understand?

The header doesn't have a way to get the back button title for previous scene, so it won't able to show a title in a custom back button.